test/tests/activitiesTest.js

```diff
@@ -187,5 +238,15 @@ module.exports = function (request, db) {
                         });
                 });
         });
+
+        it('should DELETE an activity', function (done) {
+            request.delete('/api/activities/' + idActivity)
+                .expect(200)
+                .set('X-Gleaner-User', 'Teacher1')
+                .end(function (err, res) {
+                    should.not.exist(err);
+                    done();
```

Comment:
We should check that the activity has been deleted from mongoDB.

Also, check that the PERMISSIONS have been removed from mongo associated to that activity related with kibana visualisations/dashboards and default indices IDs.

Finally,  maybe we should also check that the data sent to that activity and "processed" by realtime to finally be written to elasticsearch has been erased (erasing the session_ID index of elasticsearch?)